.vscode/settings.json

```diff
@@ -0,0 +1,47 @@
+// Recall to run the following command before even run vscode.
+// VScode will use cabal for building and one can seperately run
+// stack without any problem. The converse workflow also works.
+
+// $ cd juvix
+// $ cp cabal.hie.yaml hie.yaml
+// $ cabal build
+// $ code .
+
+{
+  "editor.tabSize": 2,
+  "editor.renderWhitespace": "trailing",
+  "haskell.formattingProvider": "ormolu",
+  "haskell.openSourceInHackage": false,
+  "haskell.openDocumentationInHackage": false,
+  "haskell.checkProject": true,
+  "haskell.manageHLS": "GHCup",
+  "haskell.serverExecutablePath": "${HOME}/.local/bin/haskell-language-server",
```

Comment:
I'm not sure if this is a good idea at all. You're forcing people to use GHCup and a particular hard-coded path to the HLS. It's not possible to change this locally because the local configuration files are version-controlled with git.